### PR TITLE
Fix the control map qemu vs test host timing issue

### PR DESCRIPTION
**Pull request by oleorhagen**

### CODE REVIEW

**Comment by oleorhagen** [comment]
Depends on: https://github.com/mendersoftware/meta-mender/pull/1866

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

File: tests/acceptance/test_update_control.py [hunk 1]
```diff
@@ -492,7 +492,7 @@ def test_update_control_maps(
                 # boot expiration mechanism.
                 connection.run("systemctl restart mender-client")
 
-            now = time.time()
+            now = qemu_system_time(connection)
```
Comment (lluiscampos) [inline tests/acceptance/test_update_control.py:495]: Once we are here, we could rename `now` to something more meaningful like `test_start_time` or `update_start_time`...

**Review by lluiscampos — APPROVED** [review]

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

File: tests/acceptance/test_update_control.py (hunk 1 above)
Comment (oleorhagen) [reply tests/acceptance/test_update_control.py:495]: Yes - agreed! Let's. I'll add a separate commit for this.

**Comment by oleorhagen** [comment]
@mender-test-bot start pipeline

**Pull request merged**